lib/rules/max-len.js

```diff
@@ -157,11 +157,14 @@ module.exports = {
          */
         function isFullLineComment(line, lineNumber, comment) {
             var start = comment.loc.start,
-                end = comment.loc.end;
+                end = comment.loc.end,
+                isFirstTokenOnLine = !line.slice(0, comment.loc.start.column).trim(),
```

Comment:
Could be - it's pretty niche (really only useful to see if the comment is the first thing on that line)